src/dotnet/ShellShim/ShellShimMaker.cs

```diff
@@ -51,17 +60,42 @@ public void EnsureCommandNameUniqueness(string shellCommandName)
             }
         }
 
+        internal void CreateConfigFile(string outputPath, string entryPoint, string runner)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Is this check really needed? You did it above before invoking this method.

Reply:
probably not needed. I added it just to be safe.